tests/verifyMysqlDashboards_test.js

```diff
@@ -50,14 +56,12 @@ Scenario(
   },
 );
 
-Scenario(
-  'PMM-T1070 - Verify link to instructions for enabling rendering images @nightly @dashboards',
-  async ({ I, dashboardPage, links }) => {
-    const metricName = 'Client Connections (All Host Groups)';
-
-    I.amOnPage(`${dashboardPage.proxysqlInstanceSummaryDashboard.url}?from=now-5m&to=now`);
+Data(urlsAndMetrics).Scenario(
+  'PMM-T1070 + 449 - Verify link to instructions for enabling rendering images @nightly @dashboards',
```

Comment:
@ElenaFedichkina is `449 ` and ID of a test case? if yes then please use full ID without `+` like: `PMM-T449`

Reply:
ok. thanks